README.md

```diff
@@ -1,23 +1,23 @@
 ![FlyByWire Simulations](https://raw.githubusercontent.com/flybywiresim/branding/1391fc003d8b5d439d01ad86e2778ae0bfc8b682/tails-with-text/FBW-Color-Light.svg#gh-dark-mode-only)
 ![FlyByWire Simulations](https://github.com/flybywiresim/branding/blob/master/tails-with-text/FBW-Color-Dark.svg#gh-light-mode-only)
 
-# FlyByWire Simulations A32NX
+# FlyByWire Simulations Aircraft
 
-[![GitHub latest release version](https://img.shields.io/github/v/release/flybywiresim/a32nx.svg?style=flat)](https://github.com/flybywiresim/a32nx/releases/latest)
-[![Github All Releases download count](https://img.shields.io/github/downloads/flybywiresim/a32nx/total.svg?style=flat)](https://github.com/flybywiresim/a32nx/releases/latest)
-[![GitHub contributors](https://img.shields.io/github/contributors/flybywiresim/a32nx.svg?style=flat)](https://github.com/flybywiresim/a32nx/graphs/contributors)
+[![GitHub latest release version](https://img.shields.io/github/v/release/flybywiresim/aircraft.svg?style=flat)](https://github.com/flybywiresim/a32nx/releases/latest)
+[![Github All Releases download count](https://img.shields.io/github/downloads/flybywiresim/aircraft/total.svg?style=flat)](https://github.com/flybywiresim/a32nx/releases/latest)
+[![GitHub contributors](https://img.shields.io/github/contributors/flybywiresim/aircraft.svg?style=flat)](https://github.com/flybywiresim/a32nx/graphs/contributors)
 
 [![Discord](https://img.shields.io/discord/738864299392630914.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/UjzuHMU)
 [![Twitter](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=twitter&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://twitter.com/FlybywireSim)
 [![YouTube](https://img.shields.io/badge/-FlyByWireSimulations-e84393?label=&logo=youtube&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://www.youtube.com/c/FlyByWire-Simulations)
 [![Facebook](https://img.shields.io/badge/-FlyByWireSimulations-e84393?label=&logo=facebook&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://www.facebook.com/FlyByWireSimulations/)
 [![Instagram](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=instagram&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://instagram.com/flybywiresim)
 
-The A32NX Project is a community-driven open source project to create a free Airbus A320neo in Microsoft Flight Simulator that is as close to reality as possible.
+The A32NX and A380X Projects are community-driven open source projects to recreate, respectively, a free Airbus A320neo and A380-800 in Microsoft Flight Simulator that are as close to reality as possible.

```

Comment:
```suggestion
The A32NX and A380X projects are community-driven open source projects to recreate, respectively, a free Airbus A320neo and A380-800 in Microsoft Flight Simulator that are as close to reality as possible.
```

Reply:
Might as well fix the weird phrasing too, and also correct the tense.
```No, they are completely free and open source, along with SimBridge and other FlyByWire projects which are publicly accessible via GitHub.```